src/components/GeomUtilsService.js

```diff
@@ -194,6 +194,18 @@ goog.provide('ga_geomutils_service');
           }
           return geom;
         };
+
+        // Simplify a geometry only if the number of coordinates exceeds
+        // nbPointsMax 
+        this.simplify = function(geom, nbPointsMax) {
+          if (!(geom instanceof ol.geom.LineString) || !nbPointsMax ||
+              geom.getCoordinates().length <= nbPointsMax) {
+            return geom;
+          }
+          // Magic number coming from service-alti
+          // https://github.com/geoadmin/service-alti/blob/master/alti/views/profile.py#L53
+          return geom.simplify(12.5);
```

Comment:
This is the simplification, but it doesn't assure the max number of points which happens here? https://github.com/geoadmin/service-alti/blob/master/alti/views/profile.py#L150

This will probably help already, so good to me to merge.

Reply:
After disacuss wthis with @loicgasser 
The profile uses the same method but with 500 instead of 1000. There is no limitations it returns all the  points which are in the geometry simplified. For example ithe profile service returns 5000 points for the geometry ulrich.txt file.

Reply:
And with a limitation  to 1000 coords the resulting profile was really not precise.

Reply:
So the request on the backend stays large?

The goal is to reduce the size of the request to the backend.

Reply:
5000 points is not large @loicgasserr says [here](https://github.com/geoadmin/mf-geoadmin3/issues/4044) that 150000 points is large.

Reply:
Probably true.

The exact number os not so relevant as the fact that we need to avoid sending too much. And as I understand the algorithm, we don't reduce the number of points sent.

So clients could still send things that will break our backends. This we need to avoid.